src/extensions/addrbar/bootstrap-table-addrbar.js

```diff
@@ -44,6 +44,8 @@ function _buildUrl (dict, url = window.location.search) {
     // 搜索name=general这种形式的字符串(&是分隔符)
     const pattern = `${key}=([^&]*)`
     const targetStr = `${key}=${val}`
+    if (val === undefined )
```

Comment:
Could be changed to the normal syntax with `{}`, but as the extension should be rewritten someday its ok.